simpegEM/FDEM/SurveyFDEM.py

```diff
@@ -16,6 +16,20 @@ class RxFDEM(Survey.BaseRx):
                     'bxi':['b', 'Fx', 'imag'],
                     'byi':['b', 'Fy', 'imag'],
                     'bzi':['b', 'Fz', 'imag'],
+
+                    'jxr':['j', 'Fx', 'real'],
```

Comment:
Yeah, I think that we should have a JH rx and a EB rx. Or we could switch between them on the fly by some multiplications right?